tests/test_smart_completion_public_schema_only.py

```diff
@@ -32,7 +32,7 @@ def completer():
     tables, columns = [], []
     for table, cols in metadata['tables'].items():
         tables.append(('public', table))
-        columns.extend([('public', table, col) for col in cols])
+        columns.extend([('public', table, col, 'text') for col in cols])
```

Comment:
Maybe add a comment here to point out that we're hardcoding the dummy column types to 'text'?